package.json

```diff
@@ -1,67 +1,66 @@
 {
-  "name": "inversify-express-utils",
-  "version": "6.4.7",
   "author": "Cody Simms",
-  "description": "Inversify wrapper for the express framework",
-  "license": "MIT",
-  "main": "lib/index.js",
-  "module": "es/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/inversify/inversify-express-utils.git"
-  },
-  "scripts": {
-    "build": "npm run build:lib && npm run build:es && npm run build:es6",
-    "build:lib": "tsc -p src/tsconfig.json",
-    "build:es": "tsc -p src/tsconfig-es.json",
-    "build:es6": "tsc -p src/tsconfig-es6.json",
-    "clean": "rm -r es es6 lib",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "pretest": "eslint ./",
-    "eslint:fix": "eslint --fix ./",
-    "publish-please": "publish-please",
-    "prepublish": "npm run build && publish-please guard",
-    "update": "updates --update --minor && (git diff-files --quiet package.json || (rimraf package-lock.json node_modules && npm install))",
-    "postupdate": "git diff-files --quiet package-lock.json || npm test"
-  },
   "bugs": {
     "url": "https://github.com/inversify/inversify-express-utils/issues"
   },
-  "homepage": "https://github.com/inversify/inversify-express-utils#readme",
-  "jsnext:main": "es/index.js",
-  "keywords": [
-    "InversifyJS",
-    "express"
-  ],
   "dependencies": {
-    "express": "4.21.1",
-    "http-status-codes": "2.3.0",
-    "inversify": "^6.0.3"
```

Comment:
`^` has to be removed in packages, you want to make sure you're locked to package versions and avoid any auto update
should be:

`6.0.3` rather than `^6.0.3`

Reply:
But we should allow any minor update from `6.0.3` not only the `6.0.3` version. If this was not a peer dependency I would agree.

Reply:
This particular item I disagree. When you have a product everything done  in that particular product has to be part of a controlled process, a process that allows you to manually update, test before update, to really make sure that the minor, or fix didn't create issue on your library. That PR for updating libraries could be automated as well via snyk, dependabot etc.

Is good practice @notaphplover. For libraries locked versions (e.g., "6.0.3") are generally preferred in dependencies for stability and predictability, especially for foundational libraries where consistency across projects is critical.

That would be my recommendation. 

Little table that usually helps on this decision

| Factor| Locked | Caret |
|--------|--------|--------|
| Stability | Ensures consistent behavior across all installs, reducing bugs due to version changes. | Allows newer patch and minor versions that might include bug fixes and optimizations, but may introduce unexpected changes. |
| Predictability in Testing | Easier to test and guarantee behavior since every environment uses the same version. | Variations in versions between installs could lead to hard-to-replicate bugs. |
| Compatibility with Upstream Fixes | Does not receive updates unless manually upgraded, meaning bug fixes or security patches may be missed. | Automatically picks up non-breaking updates, reducing maintenance overhead for keeping packages updated. |
| Security | Older locked versions might have vulnerabilities if not actively updated. | Minor updates may include security patches, but can still introduce breaking changes if assumptions differ between versions. |
| Maintenance Overhead | Requires manual updates to benefit from bug fixes or optimizations, increasing maintenance time. | Updates are automatic, saving maintenance time as long as updates are actually non-breaking. |
| User Experience | Consistent, predictable experience for library users. | Users may face version inconsistencies, but also benefit from faster access to improvements. |